radixdlt-core/radixdlt/src/main/java/com/radixdlt/network/p2p/transport/PeerChannel.java

```diff
@@ -205,6 +206,16 @@ public void channelRead0(ChannelHandlerContext ctx, byte[] buf) throws Exception
 
 	@Override
 	public void channelInactive(ChannelHandlerContext ctx) {
+		final ValidatorAddressing validatorAddresses = ValidatorAddressing.bech32("vb");
```

Comment:
this shouldn't be hardcoded to "vb", better to use injected `Addressing`, also I think what's meant here is `NodeAddressing`, not validator (if it's meant to be a validator addr, then we should change the log message)